### PR TITLE
Log recoverable updater errors as warnings

### PR DESCRIPTION
**Pull request by casey**

### CODE REVIEW

File: src/index/updater.rs [hunk 1]
```diff
@@ -229,7 +229,7 @@ impl Updater {
 
           errors += 1;
           let seconds = 1 << errors;
-          log::error!("failed to fetch block {height}, retrying in {seconds}s: {err}");
+          log::warning!("failed to fetch block {height}, retrying in {seconds}s: {err}");
```
Comment (windsok) [inline src/index/updater.rs:232]: I think this should be `log::warn`

Comment (casey) [reply src/index/updater.rs]: Perfect, fixed. Assume it's good to go otherwise? Are there any recoverable errors that should be warnings?

**Review by raphjaph — APPROVED** [review]
Looks good, wanted to do it like this as well

**Comment by casey** [comment]
Created #1528 to track any stragglers. @raphjaph this is good to merge!

**Pull request merged**